src/mono/mono/metadata/class-internals.h

```diff
@@ -1356,6 +1356,12 @@ mono_class_get_declsec_flags (MonoClass *klass);
 void
 mono_class_set_declsec_flags (MonoClass *klass, guint32 value);
 
+gint32
+mono_class_get_inlinearray_value (MonoClass *klass);
```

Comment:
```suggestion
MONO_COMPONENT_API gint32
mono_class_get_inlinearray_value (MonoClass *klass);
```

the code in src/mono/mono/component is sometimes compiled as separate shared libraries. So if they call back into the runtime, those functions need to be marked with `MONO_COMPONENT_API` (in this case, the getter, which is called from the debugger object)

Reply:
@lambdageek  done!